src/sentry/api/endpoints/group_integration_details.py

```diff
@@ -54,15 +54,17 @@ def get(self, request, group, integration_id):
             return Response(
                 {'detail': 'This feature is not supported for this integration.'}, status=400)
 
-        # TODO(jess): add create issue config to serializer
-        return Response(
-            serialize(
-                integration,
-                request.user,
-                IntegrationIssueConfigSerializer(group, action, params=request.GET),
-                organization_id=organization_id
+        try:
+            return Response(
+                serialize(
+                    integration,
+                    request.user,
+                    IntegrationIssueConfigSerializer(group, action, params=request.GET),
+                    organization_id=organization_id
+                )
             )
-        )
+        except IntegrationError as exc:
+            return Response({'detail': exc.message}, status=400)
```

Comment:
Would it be useful long term to have a test around this behavior to catch future regressions?